pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -363,14 +368,14 @@ class Build {
                                     parameters: [
                                             context.string(name: 'UPSTREAM_JOB_NUMBER', value: "${env.BUILD_NUMBER}"),
                                             context.string(name: 'UPSTREAM_JOB_NAME', value: "${env.JOB_NAME}"),
-                                            context.string(name: 'RELEASE_TAG', value: "${buildConfig.SCM_REF}"),
                                             context.string(name: 'JDK_REPO', value: jdkRepo),
                                             context.string(name: 'JDK_BRANCH', value: jdkBranch),
                                             context.string(name: 'OPENJ9_BRANCH', value: openj9Branch),
                                             context.string(name: 'LABEL_ADDITION', value: additionalTestLabel),
                                             context.string(name: 'KEEP_REPORTDIR', value: "${keep_test_reportdir}"),
                                             context.string(name: 'PARALLEL', value: parallel),
-                                            context.string(name: 'NUM_MACHINES', value: "${numMachinesPerTest}"),
+                                            context.string(name: 'USE_TESTENV_PROPERTIES', value: useTestEnv),
+                                            context.string(name: 'ADOPTOPENJDK_BRANCH', value: aqaBranch),
```

Comment:
```suggestion
                                            // We keep the ADOPTOPENJDK label for historical compatibility.
                                            context.string(name: 'ADOPTOPENJDK_BRANCH', value: aqaBranch),
```

Reply:
Add the comment to the definition of aqaBranch https://github.com/adoptium/ci-jenkins-pipelines/pull/229/files#diff-b4bb07e2f448f9ad2986d69423ad0b704dcdc4d217d947f465e6ace3fde7a4b3R320